functional-tests/src/test/groovy/FrontendCore/spec/Dashboard.groovy

```diff
@@ -71,22 +69,22 @@ class  DashboardSpec extends GebReportingSpec {
     }
 
     def "Scenario: User can view a mine"(){
-        def viewMineName = firstMineName
+        def SearchedDashboard = new Dashboard(url: "dashboard/mines?page=1&per_page=25&search=MINETEST")
```

Comment:
Nice 👍 

Do we have a constant for our test mine name? If we could use that here instead of a hardcoded string, that would probably be best

Reply:
fixed

Reply:
Now running through the pipeline, fingers crossed